src/main/views/search.njk

```diff
@@ -45,7 +45,7 @@
                 {% if noResultsError or invalidInputError %}
                     <span class="govuk-error-message">Enter a court name, town or city</span>
                 {% endif %}
-                <div class="{{ inputClasses }}" id="search-input-container"></div>
+                <div class="{{ inputClasses }}" id="search-input"></div>
```

Comment:
we have 2 elements with the same id now, youll have to either change the name of this again or change the id in the autcomplete